Content.Server/GameObjects/EntitySystems/Atmos/GasTileOverlaySystem.cs

```diff
@@ -60,13 +61,33 @@ internal sealed class GasTileOverlaySystem : SharedGasTileOverlaySystem, IResett
 
         private AtmosphereSystem _atmosphereSystem = default!;
 
+        private int _thresholds;
+
         public override void Initialize()
         {
             base.Initialize();
 
             _atmosphereSystem = Get<AtmosphereSystem>();
             _playerManager.PlayerStatusChanged += OnPlayerStatusChanged;
             _mapManager.OnGridRemoved += OnGridRemoved;
+            var configManager = IoCManager.Resolve<IConfigurationManager>();
+            var tickRate = configManager.GetCVar(CCVars.NetGasOverlayTickRate);
+            if (tickRate > 0.0f)
+            {
+                _updateCooldown = 1 / tickRate;
+            }
+            else
+            {
+                _updateCooldown = float.MaxValue;
+            }
+
+            _updateRange = configManager.GetCVar(CVars.NetMaxUpdateRange) + RangeOffset;
+
+            configManager.OnValueChanged(CCVars.NetGasOverlayTickRate, value => _updateCooldown = value > 0.0f ? 1 / value : float.MaxValue);
+            configManager.OnValueChanged(CVars.NetMaxUpdateRange, value => _updateRange = value + RangeOffset);
```

Comment:
The configuration manager's `OnValueChanged` has a parameter to invoke the lambda immediately so you don't need to duplicate the code and manually get the tick rate, range, etc